DOCUMENTATION.md

```diff
@@ -340,15 +340,19 @@ The personalisation argument always contains the following required parameters f
 
 - `address_line_1`
 - `address_line_2`
-- `postcode` (this needs to be a real UK postcode)
+– `address_line_7` this needs to be a real UK postcode or the name of a country outside the UK
+
+Notify uses `address_line_7` to check for international addresses. This is so we can charge you the correct postage.
+
+The `postcode` personalisation argument has been replaced. If your template still uses `postcode`, Notify will treat it as `address_line_7`.
```

Comment:
"treat it as"?
"read it as"?
"map it to"?

What's the best way to describe this for our users?